custom_components/tuya_local/devices/immax_neolite_weatherstation.yaml

```diff
@@ -433,13 +431,15 @@ entities:
           - dps_val: angle
             value: Compass angle
   - entity: sensor
-    name: Power status
+    name: Battery
```

Comment:
As above, it is better not to rename. In this case it seems the meaning is also changed, so it can be an exception, but use "Outdoor battery".

Is there also other info encoded in the same binary field? In that case, retaining the raw value so it can be extracted would also be useful, but since raw binary data is not really useful as a sensor, making it an attribute of the newly renamed sensor would be OK.